vector/src/main/res/values/strings.xml

```diff
@@ -2812,6 +2812,8 @@
     <string name="direct_room_profile_section_more_leave">Leave</string>
     <string name="room_profile_leaving_room">"Leaving the room…"</string>
 
+    <string name="room_member_override_nick_color">Override nick color</string>
```

Comment:
is the `nick` terminology used in other places, or should this be `display name`?

Reply:
Nick is used in the slash command `/myroomnick` but nowhere else AFAIK. The color will be used to tint the display name but also for the background of the default avatar. So not only the display name.
Maybe using "Override user color" is fine. Please vote with 👍 and 👎 !

Reply:
no strong opinions from me on this :smile: maybe one for product cc @daniellekirkwood